Multisig/UI/ClaimToken/ReviewClaimSafeTokenTransactionViewController.swift

```diff
@@ -8,25 +8,27 @@
 
 import UIKit
 import SwiftCryptoTokenFormatter
+import Solidity
 
 class ReviewClaimSafeTokenTransactionViewController: ReviewSafeTransactionViewController {
     private var stepLabel: UILabel!
     
     var stepNumber: Int = 4
     var maxSteps: Int = 4
 
-    var onSuccess: (() -> ())?
+    var amount: Sol.UInt128!
+    var claimData: ClaimingAppController.ClaimingData!
+    var timestamp: TimeInterval!
+    var selectedGuardian: Guardian?
+    var selectedCustomAddress: Address?
 
-    private var guardian: Guardian!
-    private var amount: String!
-    convenience init(safe: Safe, guardian: Guardian, amount: String) {
```

Comment:
Isn't it better to keep this init method?

Reply:
👍